xdf.cpp

```diff
@@ -659,24 +501,44 @@ void Xdf::resample(int userSrate)
                 double* inbuf = new double[row.size()];
                 double* outbuf = new double[OUT_BUF_SIZE];
 
-
-                std::copy(row.begin(), row.end(), inbuf);
-
-                read = row.size();
-
+                // Fill inbuf with the numeric values from the row
+                for(auto& val : row) {
+                    std::visit([&inbuf, &read](auto&& arg) {
```

Comment:
These probably would be more readable with `std::transform` together with template.

https://cplusplus.com/reference/algorithm/transform/